components/support/base/src/main/java/mozilla/components/support/base/observer/ObserverRegistry.kt

```diff
@@ -44,18 +37,14 @@ class ObserverRegistry<T> : Observable<T> {
         val lifecycleObserver = LifecycleBoundObserver(
                 owner,
                 registry = this,
-                observer = observer)
+                observer = observer,
+                autoPause = autoPause)
 
         lifecycleObservers[observer] = lifecycleObserver
 
         owner.lifecycle.addObserver(lifecycleObserver)
     }
 
-    /**
-     * Registers an observer to get notified about changes.
-     *
-     * The observer will automatically unsubscribe if the provided view gets detached.
-     */
```

Comment:
We remove all the kdoc here because the method is documented in the interface? Do you know how dokka will handle that?

Reply:
Yes, exactly. JavaDocs and Dokka will correctly show the docs on the implementations as well. IDEs also. So, we don't need to repeat the docs on the implementation.

Reply:
Nice :)